packages/context-editor/src/style.css

```diff
@@ -76,13 +76,24 @@
 .ProseMirror ul,
 .ProseMirror ol,
 .ProseMirror li,
-.ProseMirror section {
+.ProseMirror section,
+.ProseMirror table,
```

Comment:
I think this should be like this. Otherwise the atrribute panel  is not in the correct place.

```suggestion
.ProseMirror .tableWrapper,
```

together with the removal of the left-border on the table below, it goes from this

![image](https://github.com/user-attachments/assets/f091f4d7-27cc-41d5-ab55-625e5de1e622)


to this

![image](https://github.com/user-attachments/assets/9c8058d9-7120-490a-a60d-d8892bce9767)